migrations/manual/educandu-2021-12-14-01-migrate-cdn-resources-once-more.spec.js

```diff
@@ -0,0 +1,110 @@
+import { aggregateSectionUrls } from './educandu-2021-12-14-01-migrate-cdn-resources-once-more.js';
+
+const imageContent1 = {
+  sourceType: 'internal',
+  sourceUrl: 'media/image-content-1.jpg'
+};
+const imageContent2 = {
+  sourceType: 'external',
+  sourceUrl: 'media/image-content-2.jpg'
+};
+const videoContent1 = {
+  sourceType: 'internal',
+  sourceUrl: 'media/video-content-1.mov'
+};
+const videoContent2 = {
+  sourceType: 'external',
```

Comment:
There's also the type "youtube" just for info. I think it would be overkill to add it to the tests, but maybe we could take the chance to modify the script to not populate them in the cdn resources when you have an URL marked as "internal" but it has the value of https://youtube/etwas. I think we have a couple of documents that have this problem and it was causing some import issues (at least on local, the real S3 doesn't seem to care).

Reply:
We could do that, sure, but that would not solve the problem unless we also don't allow adding external URLs when 'internal' is selected. So new such cases will be introduced.
We should make a ticket for that, fix the code and add a dedicated migrations script to correct such cases.

Reply:
https://educandu.atlassian.net/browse/EDU-187